Gemfile

```diff
@@ -42,6 +42,8 @@ gem "tzinfo-data", platforms: %i[ mingw mswin x64_mingw jruby ]
 # Reduces boot times through caching; required in config/boot.rb
 gem "bootsnap", require: false
 gem "devise"
+gem 'rails_admin'
+gem "sassc-rails"
 # Use Sass to process CSS
 # gem "sassc-rails"
```

Comment:
Remove the comment for `# sassc-rails` since you included the gem above in the line 46.

Reply:
True, on it!